editing/authoring-rules.adoc

```diff
@@ -1,408 +1,113 @@
-= Executing Services
+= Business Rules with Drools 
```

Comment:
This should probably be drl instead of Drools. We're trying to avoid using the community project names.